### PR TITLE
Use more resources - these builds are too slow

### PR DESCRIPTION
**Pull request by timja**
13 minutes to install dependencies - [infra.ci.jenkins.io](https://infra.ci.jenkins.io/job/plugin-site/job/PR-958/11/console)  (200m cpu, 256mb memory)
2m30 - [ci.jenkins.io on VM](https://ci.jenkins.io/blue/organizations/jenkins/Infra%2Fplugin-site/detail/PR-958/14/pipeline/19)

11m 34s with 1 cpu 2gb memory
2m 9s with 2 cpu 4gb memory

### CODE REVIEW

File: Jenkinsfile_k8s [hunk 1]
```diff
@@ -55,6 +55,11 @@ spec:
     - name: "jnlp"
       image: "jenkinsciinfra/builder"
       imagePullPolicy: Always
+      resources:
+        limits: {}
+        requests:
+          memory: "2Gi"
+          cpu: "1"
```
Comment (timja) [inline Jenkinsfile_k8s]:
```suggestion
          memory: "4Gi"
          cpu: "2"
```

File: Jenkinsfile_k8s [hunk 2]
```diff
@@ -55,6 +55,11 @@ spec:
     - name: "jnlp"
       image: "jenkinsciinfra/builder"
       imagePullPolicy: Always
+      resources:
+        limits: {}
```
Comment (dduportal) [inline Jenkinsfile_k8s:59]: Is there a reason to not put limits? (I see a lot for setting limits in term of hosting tbh)

**Comment by timja** [comment]
Still quite a lot slower than ci.jenkins.io

ci.jenkins.io is `10m 22s`

infra.ci.jenkins.io is still going, although it is saying 503 atm

**Comment by dduportal** [comment]
> Still quite a lot slower than ci.jenkins.io
> 
> ci.jenkins.io is `10m 22s`
> 
> infra.ci.jenkins.io is still going, although it is saying 503 atm

I gues it's the kind of worker defined for the temp cluster: these are slow and unreliable (but cheap) instances.
Feel free to manually change the instance size of the user pool to get better perfs.

**Comment by timja** [comment]
Will fiddle with it a bit later, but right now it’s painfully slow

**Comment by halkeye** [comment]
I'll admit I never have truely understood these resources blocks. I understand the concept but not the values. So I'm signing off on whatever timja sets it too

**Comment by timja** [comment]
Second build completed in 14m total vs ci.jenkins.io 10m probably good enough for now.

**Comment by zbynek** [comment]
btw. could one somehow avoid building PRs on 2 different CI instances? Maybe the infra.ci Jenkinsfile could be only responsible for testing & publishing  the master branch, the other Jenkinsfile for testing PRs.

**Comment by timja** [comment]
> btw. could one somehow avoid building PRs on 2 different CI instances? Maybe the infra.ci Jenkinsfile could be only responsible for testing & publishing the master branch, the other Jenkinsfile for testing PRs.

infra.ci is publishing to netlify here for the preview deployment.

The tests could be skipped on infra.ci PRs but the tests are only 30 seconds of the build.

the last build was:

- `yarn install` 2min
- `yarn build` 5min20
- `yarn lint; yarn test` 37s
- `netlify deploy` 5m47s

**Comment by halkeye** [comment]
honestly we could skip ci.jenkins.io, its just nicer for non infra people to see the results of the build

**Pull request merged**